packages/luna/src/components/TutorProfile/TutorProfileCard/TutorProfileCard.tsx

```diff
@@ -38,6 +39,7 @@ export const TutorProfileCard: React.FC<{
   loading,
   error,
   retry,
+  desktop = false,
```

Comment:
If we went with this approach, it will be hard to customize in the future. For example, if we added a new prop and called it `tablet`. 

```
desktop?: boolean;
table?: boolean; 
```

The component interface will give the assumption that you can provide both props at the same time! Also, this will left us with unexpected behaviour. 

A better interface, will reflect the real-world use cases. 

```ts
mq: 'default' | 'sm' | 'md'
```